src/verify.js

```diff
@@ -0,0 +1,68 @@
+const utils = require('./utils');
+
+const verifyViews = async (dbName, numDocs) => {
+  if (!numDocs) {
+    return true;
+  }
+
+  const url = utils.getUrl(`${dbName}/_design_docs`, false, 'include_docs=true');
+  const response = await utils.request({ url });
+
+  let viewsIndexed = false;
+  let viewsChecked = false;
+
+  for (const { doc: ddoc } of response.rows) {
+    if (!ddoc || !ddoc.views) {
+      continue;
+    }
+
+    const views = Object.keys(ddoc.views);
+    for (const view of views) {
+      viewsChecked = true;
+      const url = utils.getUrl(`${dbName}/${ddoc._id}/_view/${view}`, false, 'stale=ok&limit=0');
+      const viewResponse = await utils.request({ url });
+
+      if (viewResponse.total_rows > 0) {
+        // at least one view being indexed means that the .shard folders for the database were moved successfully
+        // some views might not index any documents
+        viewsIndexed = true;
```

Comment:
Why not short circuit here? ie...

```suggestion
        return true;
```

Reply:
I did short circuit initially. Then I wanted to test all views (I did some manual custom tests, for good measure) and forgot to revert the code back.

Reply:
Ah, I can see that I later added unit tests to check I go over every view too!